api/models.py

```diff
@@ -844,8 +869,6 @@ class FirstStage(models.Model):
     launcher = models.ForeignKey(Launcher, related_name='firststage', on_delete=models.CASCADE)
     rocket = models.ForeignKey(Rocket, related_name='firststage', on_delete=models.CASCADE)
 
-    # Not a property but keeping around as a helper
-
     @property
     def previous_flight(self):
```

Comment:
Any reason this is added back as a property?

Reply:
Shouldnt cause any problems but no - not sure why it happened.

Reply:
Yeah no problems, just unnecessary